doc/sphinx/conf.py

```diff
@@ -76,7 +76,9 @@
 version = '0.20'
 
 # The Documentation full version, including alpha/beta/rc tags. Some features
-# available in the latest code will not necessarily be documented first
+# available in the latest code will not necessarily be documented first.
+# rc syntax may be tagged; this documentation supports various rc-naming conventions 
+
 release = '0.20.0'
```

Comment:
0.21.0 (and version above)

Reply:
https://github.com/NervanaSystems/ngraph/blob/leona/be_developer_guide/doc/sphinx/source/conf.py